pom.xml

```diff
@@ -72,7 +77,9 @@ Bundle-SymbolicName: ${project.artifactId}.nodeps
                             <bnd><![CDATA[
 -includeresource:@jakarta.activation-[0-9.]*(-SNAPSHOT)?.jar
 # version automatically determined from the MANIFEST.MF of the jakarta.activation.jar
+Bundle-Activator: org.apache.sling.javax.activation.internal.Activator
```

Comment:
Why was the annotation from https://github.com/apache/sling-org-apache-sling-javax-activation/pull/4/files#diff-647f7f295a3a772b13eaaa75470004f7L52 not working?

Reply:
The nodeps bundle is a framework extension fragment. That one needs a different header: `ExtensionBundle-Activator` instead of `Bundle-Activator`, so I couldn't use the annotation for the 2 ways in which the activator was used.

The actual interface to be implemented by the Extension Bundle Activator is the same as the normal Bundle Activator.